.github/ISSUE_TEMPLATE/squad.md

```diff
@@ -0,0 +1,21 @@
+---
+name: Squad
+about: A new squad iniative to be added on the CHT Roadmap.
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+### Overview
+<!-- What is the objective of this squad? -->
```

Comment:
Maybe something like "Remember, this should be a well defined, short term objective. It's not "meet to talk about backup", but instead, "implement a native backup feature allowing backup to an S3 compatible back end" "